plugins/common/trace_sampling_client/trace_sampling_client.go

```diff
@@ -74,13 +74,18 @@ func (t *Client) Start() {
 	}()
 }
 
-func (t *Client) ShouldTrace(host string) bool {
+func (t *Client) ShouldTrace(host, port string) bool {
 	t.lock.RLock()
 	defer t.lock.RUnlock()
 
 	if len(t.Hosts) == 0 {
 		return false
 	}
+
+	if port != "" {
```

Comment:
Apparently, it's also possible to have `*` for the port.
Need to align with @aduminuc about that. (see https://github.com/openclarity/apiclarity/blob/devel-global-specs/backend/pkg/modules/internal/core/core.go#L116)

Reply:
Not sure, Is it supported also in Trace Sampling Manager?

Reply:
After discussion with @aduminuc, it may be supported in the future. And is already supported in APIClarity.
So, no need to take that into accound right now.

All good.